### PR TITLE
bump steem-js version to 72a36c8

### PR DESCRIPTION
**Pull request by valzav**
https://github.com/steemit/steem-js/compare/ec4439c...72a36c8

### CODE REVIEW

**Comment by goldibex** [comment]
My changes to `steem-js` were made to introduce appbase support on the client. Appbase itself is behind an option, but I did make changes that affect non-appbase Websocket clients, mostly rationalization of portions of the codebase badly affected by technical debt. Tests (including new ones I introduced specifically _for_ the appbase support) passed on those changes which is why I felt comfortable releasing them.

File: package.json [hunk 1]
```diff
@@ -126,7 +126,7 @@
     "sequelize-cli": "^2.3.1",
     "speakingurl": "^9.0.0",
     "sqlite3": "^3.1.8",
-    "steem": "github:steemit/steem-js#ec4439c",
+    "steem": "github:steemit/steem-js#72a36c8",
```
Comment (sneak) [inline package.json]: we should be using the full sha of the commit, not the short one.

**Comment by goldibex** [comment]
Can you describe in the PR issue the reason for the version bump?

**Comment by roadscape** [comment]
Looking at the diff, the biggest change was a refactor with the addition of appbase switcher. 
Lgtm but good to get more eyes on it: https://github.com/steemit/steem-js/commit/1ebe788987d395af5fd8c80c1bbf06e24614a86c

Everything else is straightforward; tx expiration changes from 60 to 600s and tapos ref block is changed to use last_irreversible. I've left a note to revert both of these changes in the future (https://github.com/steemit/steem-js/issues/265). 
These changes will help in 2 conditions: (1) tx's not propagating within 60s, and (2) nodes being behind by up to 30 secs in some cases, when currently steem-js code requires they are at most 9 seconds (3 blocks) behind.

**Pull request merged**